realm-transformer/src/main/groovy/io/realm/transformer/RealmTransformer.groovy

```diff
@@ -278,4 +288,23 @@ class RealmTransformer extends Transform {
         return merged;
     }
 
+    // Use reflection to get the AndroidBuild from Android plugin for getting the path to android.jar
+    // There is no official way to get the path to android.jar for transform.
```

Comment:
The transform page ask people to report issue to adt-dev group, i made a post there, but it just disappeared... I don't know why it is not moderated ...